awpy/plot/plot.py

```diff
@@ -271,11 +340,21 @@ def heatmap(
         map_name (str): Name of the map to plot.
         points (List[Tuple[float, float, float]]): List of points to plot.
         method (Literal["hex", "hist", "kde"]): Method to use for the heatmap.
+        is_lower (optional, bool): If set to False, will NOT draw lower-level
+            points. If True will draw only lower-level points on the
+            lower-level minimap. Defaults to False.
+        ignore_extreme_points (bool, optional): If set to True, will ignore
+            points that are outside of the map graphic. If set to False, will
+            draw those points, changing the resolution of the ouput image.
+            Defaults to False.
         size (int, optional): Size of the heatmap grid. Defaults to 10.
         cmap (str, optional): Colormap to use. Defaults to 'RdYlGn'.
         alpha (float, optional): Transparency of the heatmap. Defaults to 0.5.
         vary_alpha (bool, optional): Vary the alpha based on the density. Defaults
             to False.
+        vary_alpha_range (List[float, float], optional): The min and max transparency
```

Comment:
I'm a bit unclear as to what this parameter is doing -- do you mind explaining again? I realize I don't quite remember what I wrote 🤣

Reply:
Previously (before I added `vary_alpha_range`) when `vary_alpha` was set to `True`, it would plot the densest points with alpha equal to the `alpha` argument and the least dense points with `alpha = 0`, and anything in between would be assigned linearly between `0` and `alpha`.

`vary_alpha_range` lets the user set a custom range (instead of `0` and `alpha`).

Reply:
I now realize that `vary_alpha_range` might be a confusing name - it was intended to sound like "the range of argument `vary_alpha`", but it sounds like, "vary the alpha's range".

Can't think of anything better though - I considered `alpha_range`, but it's even more ambiguous :D

Reply:
I see. If an alpha range is specified, should we assume the user wants to vary_alpha? This may allow us to remove one arg. If so, then I think having `alpha_range` is a fine name, as long you add just a tiny bit more explanation in the docstring!

Reply:
Great idea!